### ISSUE TITLE
Github Project organization and project documentation page required

### ISSUE THREAD

**Issue by prassee** [cvogt/cbt#355]
I would like to see a separate github organization and docmentation page for CBT. As of now its under cvogt github handler.

**Comment by cvogt** [comment]
Yep. Let's aim for that in the context of 1.0. I already secured this github org a while back https://github.com/scala-cbt
